dirsrvtests/tests/suites/ds_logs/ds_logs_test.py

```diff
@@ -1369,9 +1369,47 @@ def fin():
         log.info('Restore dse.ldif')
         topology_st.standalone.stop()
         shutil.copy(dse_ldif + '.correct', dse_ldif)
+        topology_st.standalone.start()
 
     request.addfinalizer(fin)
 
+
+def test_errorlog_buffering(topology_st, request):
+    """Test log buffering works as expected when on or off
+
+    :id: 324ec5ed-c8ec-49fe-ab20-8c8cbfedca41
+    :setup: Standalone Instance
+    :steps:
+        1. Set buffering on
+        2. Reset logs and restart the server
+        3. Check for logging that should be buffered (not found)
+        4. Disable buffering
+        5. Reset logs and restart the server
+        6. Check for logging that should be found
+    :expectedresults:
+        1. Success
+        2. Success
+        3. Success
+        4. Success
+        5. Success
+        6. Success
+    """
+
+    # Configure instance
+    inst = topology_st.standalone
+    inst.config.replace('nsslapd-errorlog-logbuffering', 'on')
+    inst.deleteErrorLogs(restart=True)
+
+    time.sleep(1)
+    assert not inst.ds_error_log.match(".*connection_table_new.*")
```

Comment:
In case the log message change (no more connection_table_new) we may use what is in log_write_title ("-Directory/")

Reply:
> In case the log message change (no more connection_table_new) we may use what is in log_write_title ("-Directory/")

It needs to be logging text that happens further down after startup, so I can't use the header in this case.  Yeah it's weird how the error log works - some initial text always gets flushed right away.  So we need to check for text that gets logged "later" which makes the buffering more noticeable and testable.  

Anyway I will see what other "text" I can check for that is is more stable.